src/entropies/direct_entropies/order_statistics/order_statistics.jl

```diff
@@ -0,0 +1,17 @@
+"""
+    ith_order_statistic(ex, i::Int, n::Int = length(x))
+
+Return the i-th order statistic from the order statistics `ex`, requiring that
```

Comment:
You call `ex` the order statistics (still don't know what this means), but in fact when this function is called, I can see that `ex` is actually `sort(x)`. So `ex` is a "sorted vector". Why do you call it "order statistics"? 

(one word per concept)

Reply:
Addressed in the comment above. We can just refer to k-th sorted value, or something like that, and skip technical jargon.